webapp/pages/profile/_id/_slug.vue

```diff
@@ -238,8 +238,10 @@
         <div
           v-if="hasMore"
           v-infinite-scroll="showMoreContributions"
-          infinite-scroll-disabled="$apollo.loading"
-          infinite-scroll-distance="10"
+          :infinite-scroll-immediate-check="true"
```

Comment:
what is the purpose of adding this?
doesn't this default to true anyways?
https://sroze.github.io/ngInfiniteScroll/documentation.html
or am I looking at a different library??

Reply:
Right! I double-checked and it seems it is the default per documentation. I wonder why this fixed some our cypress tests earlier. Maybe another change or the documentation is just wrong.

Reply:
Done, ping @mattwr18